### PR TITLE
fix some typos; improve argument references

### PR DESCRIPTION
**Pull request by Habbie**
### Short description
Minor dnsdist doc improvements.

### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled and tested this code
- [x] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added regression tests
- [ ] added unit tests
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: pdns/README-dnsdist.md [hunk 1]
```diff
@@ -130,28 +130,28 @@ To force a server down, try:
...
 of `newServer` is set to true, a response will only be considered valid if
 its RCODE differs from NXDomain, ServFail and Refused.
 The number of health check failures before a server is considered down is
-configurable via the`maxCheckFailures` parameter, defaulting to 1.
+configurable via the `maxCheckFailures` parameter, defaulting to 1.
 The `CD` flag can be set on the query by setting `setCD` to true.
 
 ```
-newServer({address="192.0.2.1", checkType="AAAA", checkName="a.root-servers.net.", mustResolve=true})
+newServer({address="192.0.2.1", checkType="AAAA", checkName="a.root-servers.net.", mustResolve=true}) -- FIXME: what is this code segment for?
```
Comment (rgacogne) [inline pdns/README-dnsdist.md]: I guess the intent was to document the `checkType`, `checkName` and `mustResolve` parameters described above

File: pdns/README-dnsdist.md [hunk 2]
```diff
@@ -214,16 +214,16 @@ The supported values for `source` are:
 
  * an IPv4 or IPv6 address, which must exist on the system
  * an interface name
- * an IPv4 or IPv6 address followed by '@' then an interface name
+ * an IPv4 or IPv6 address followed by `@' then an interface name
```
Comment (rgacogne) [inline pdns/README-dnsdist.md]: Looks like mixing a back quote with a regular one

Comment (Habbie) [inline pdns/README-dnsdist.md]: Oops, I changed that to `` but it became ugly and then I only half changed it back.

File: pdns/README-dnsdist.md (hunk 1 above)
Comment (Habbie) [inline pdns/README-dnsdist.md]: Ah yes, I did not look farther than the `CD` description which is not in the example.

**Comment by Habbie** [comment]
Ready.

File: pdns/README-dnsdist.md [hunk 3]
```diff
@@ -867,12 +867,12 @@ pc = newPacketCache(10000, 86400, 0, 60, 60)
 getPool(""):setCache(pc)
 ```
 
-The first parameter is the maximum number of entries stored in the cache, and is the
-only one required. All the others parameters are optional and in seconds.
-The second one is the maximum lifetime of an entry in the cache, the third one is
+The first parameter (10000) is the maximum number of entries stored in the cache, and is the
+only one required. All the others parameter are optional and in seconds.
```
Comment (rgacogne) [inline pdns/README-dnsdist.md]: Shouldn't that be 

> all the other parameters

instead?

Comment (Habbie) [inline pdns/README-dnsdist.md]: fixed

**Pull request merged**